includes/definitions.inc.php

```diff
@@ -1058,7 +1058,7 @@
 
 // EMC FlareOS
 $os = 'flareos';
-$config['os'][$os]['text'] = 'EMC CLARiiON';
+$config['os'][$os]['text'] = 'FLARE OS';
```

Comment:
That's fine for the os name, this is just a descriptive piece of text so EMC FLARE OS (or EMC Flare OS) would be better.